data/people.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "id": 61,
+    "type": "Student",
+    "age": "Unknown",
+    "name": "Unknown"
+  },
+  {
+    "id": 40,
+    "type": "Teacher",
+    "age": "Unknown",
+    "name": "Unknown"
+  },
+  {
+    "id": 30,
+    "type": "Student",
+    "age": "ZAZA",
+    "name": "Unknown"
+  },
+  {
+    "id": 12,
+    "type": "Student",
+    "age": 23,
+    "name": "ZEZE"
+  },
+  {
+    "id": 49,
+    "type": "Student",
+    "age": 222,
+    "name": "PMPM"
+  },
+  {
+    "id": 98,
+    "type": "Student",
+    "age": 22,
+    "name": "ZZZZ"
+  },
+  {
+    "id": 73,
+    "type": "Student",
+    "age": 222,
+    "name": "PHILIPPE"
+  },
+  {
+    "id": 18,
+    "type": "Student",
+    "age": 23,
+    "name": "ZE"
+  },
+  {
+    "id": 34,
+    "type": "Student",
+    "age": 111,
+    "name": "MZMMZ"
+  },
+  {
+    "id": 86,
+    "type": "Teacher",
+    "age": 23,
+    "name": "ETET"
+  },
+  {
+    "id": 100,
+    "type": "Student",
+    "age": 122,
+    "name": "AAA"
+  },
+  {
+    "id": 74,
+    "type": "Student",
+    "age": 222,
+    "name": "ZZZZ"
+  },
+  {
+    "id": 76,
+    "type": "Student",
+    "age": 122,
+    "name": "ZZ"
+  }
+]
```

Comment:
> - [ ] From your input saved data, I suggest you add the `"parent_permission": boolean (true or false)` for each class or type (Student or Teacher) and also `"specialization": ""` for the teacher class since it is part of the data that you receive as an input.👍
> 
> This is what I mean by the example below:
> 
>  ```
> [
> {
>     "id": 85,
>     "type": "Student",
>     "age": 99,
>     "name": "Bob",
>     "parent_permission": true
>   },
>   
>   {
>     "id": 8,
>     "type": "Teacher",
>     "age": 878,
>     "name": "jones",
>     "parent_permission": false,
>     "specialization": "Computer Science"
>   }
> ]
> ```